### ISSUE TITLE
Quoted key become bare key on Load

### ISSUE THREAD

**Issue by Crazy3lf** [pelletier/go-toml#290]
**Describe the bug**
Quoted key get unquoted on Load. When the affected key contains invalid character, it becomes an invalid bare key.

**To Reproduce**
Code: 
```Go
func main() {
	tomlString :=
		`[table]
"127.0.0.1" = "value"
"127.0.0.1:8028" = "value"
"character encoding" = "value"
"ʎǝʞ" = "value"`

	t1, _ := toml.Load(tomlString)
	s, _ := t1.ToTomlString()
	println(s)
	_, err := toml.Load(s)
	if err != nil {
		panic(err)
	}
}
```

Output:
```
[table]
  127.0.0.1 = "value"
  127.0.0.1:8028 = "value"
  character encoding = "value"
  ʎǝʞ = "value"

panic: (4, 3): parsing error: keys cannot contain : character
```

**Expected behavior**
According to [this section of TOML v0.4.0 documentation](https://github.com/toml-lang/toml/blob/master/versions/en/toml-v0.4.0.md#table), these key are valid. 

**Versions**
 - go-toml: v1.4.0 (728039f679cbcd4f6a54e080d2219a4c4928c546)
 - go: go1.12.6
 - operating system: windows/amd64

**Additional context**
I first encounter this problem when I WriteTo a file and try to Load from the file again and fail. When I inspected the file, I found out that all the quoted key have been overwritten by their evil bare-key clone.
Please correct me if I am wrong or there is other way of doing what I am trying to do. 
This is also the first time I open an Issue, I apologize for any error on the format.

**Comment by pelletier** [comment]
Thanks for the report! That indeed looks like a bug. `ToTomlString` should absolutely quote keys that do not fit the bare format (`A-Za-z0-9_-`).

**Comment by pelletier** [comment]
@Crazy3lf Merged #291, which should fix this bug. Do you mind trying the latest `master` and confirm it worked?

**Comment by Crazy3lf** [comment]
Yes, it works great! Thank you.
